server/externs/crypto-js.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * @fileoverview crypto-js library extern.
+ */
+
+/** @const */
+var CryptoJS = {};
+
+/**
+ * @param {string} message
+ * @return {!Object}
```

Comment:
Well, the easiest way is to just open your browser and run CryptoJS.SHA256(anything);